pkg/agents/team6/config.go

```diff
@@ -95,8 +94,18 @@ func (a *CustomAgent6) Run() {
 	a.TakeFood(foodAmount)
 	a.Log("Team 6 took:", infra.Fields{"foodTaken": foodAmount})
 
-	msg := *messages.NewBaseMessage(a.Floor())
+	msg := messages.NewResponseMessage(a.Floor(), true)
 	a.SendMessage(1, msg)
 	a.Log("Team 6 sent message:", infra.Fields{"floor": a.Floor(), "messageType": msg.MessageType()})
 
 }
+
+func (a *CustomAgent6) HandleAskHP(msg infra.AskMessage)                    {}
+func (a *CustomAgent6) HandleAskFoodTaken(msg infra.AskMessage)             {}
+func (a *CustomAgent6) HandleAskIntendedFoodTaken(msg infra.AskMessage)     {}
+func (a *CustomAgent6) HandleRequestLeaveFood(msg infra.RequestMessage)     {}
+func (a *CustomAgent6) HandleRequestTakeFood(msg infra.RequestMessage)      {}
+func (a *CustomAgent6) HandleResponse(msg infra.ResponseMessage)            {}
+func (a *CustomAgent6) HandleStateFoodTaken(msg infra.StateMessage)         {}
+func (a *CustomAgent6) HandleStateHP(msg infra.StateMessage)                {}
+func (a *CustomAgent6) HandleStateIntendedFoodTaken(msg infra.StateMessage) {}
```

Comment:
```suggestion
```
These are already implemented in base agent so unless the team is using the handler there is no need to override with an empty function